runtime/src/main/scala/akka/grpc/GrpcClientSettings.scala

```diff
@@ -284,6 +295,28 @@ final class GrpcClientSettings private (
   def withBackend(value: String): GrpcClientSettings =
     copy(backend = value)
 
+  /**
+   * Scala API: Set this to a duration to trigger periodic refresh of the resolved endpoints, evicting cached entries
+   * if the discovery mechanism supports that. The default value `None` disables periodic refresh and instead
+   * only does refresh when the client implementation decides to.
+   *
+   * Currently only supported by the Netty client backend.
+   */
+  @ApiMayChange
+  def withDiscoveryRefreshInterval(refreshInterval: Option[FiniteDuration]): GrpcClientSettings =
```

Comment:
should this be plain FiniteDuration instead of Option? starting by default as "off" and can then be set with this?

Reply:
Hmm, yeah, maybe cleaner. But then otoh it means no way to disable it for one service if you have set a default.